gentle_web_align.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Apr 21 09:06:37 2018
+Phoneme alignment and conversion in HTK-style label file using Web-served Gentle
+This works on any type of english dataset.
+This allows its usage on Windows (Via Docker) and external server.
```

Comment:
Yep, and also because Gentle is python 2 compatible only, while this repo is python3 compatible.

In addition,  if we use server-based Gentle, we can also use external server.